controlpanel/settings/common.py

```diff
@@ -0,0 +1,453 @@
+        "django": {
+            "handlers": ["console"],
+            "level": os.environ.get("LOG_LEVEL", "DEBUG"),
+        },
+        "gunicorn": {
+            "handlers": ["console"],
+            "level": "WARNING",
+        },
+        "kubernetes": {
+            "handlers": ["console"],
+            "level": "WARNING",
+        },
+        "mozilla_django_oidc": {
+            "handlers": ["console"],
+            "level": "WARNING",
+        },
+        "urllib3": {
+            "handlers": ["console"],
+            "level": "WARNING",
+        },
+        "uvicorn": {
+            "handlers": ["console"],
+            "level": "WARNING",
+        },
+    },
+}
+
+
+# -- Sentry error tracking
+
+if os.environ.get("SENTRY_DSN"):
+    RAVEN_CONFIG = {
+        "dsn": os.environ.get("SENTRY_DSN", ""),
+        "environment": os.environment.get("ENV", "dev"),
+        "ignore_exceptions": [],
+    }
+    if "shell" in sys.argv or "shell_plus" in sys.argv:
+        RAVEN_CONFIG["ignore_exceptions"] = ["*"]
+
+else:
+    INSTALLED_APPS.remove("raven.contrib.django.raven_compat")
+
+
+# -- Static files
+
+STATICFILES_FINDERS = [
+    "django.contrib.staticfiles.finders.FileSystemFinder",
+    "django.contrib.staticfiles.finders.AppDirectoriesFinder",
+    "sass_processor.finders.CssFinder",
+]
+
+COMPRESS_JINJA2_GET_ENVIRONMENT = environment
+
+SASS_PROCESSOR_AUTO_INCLUDE = False
+SASS_PROCESSOR_ROOT = STATIC_ROOT
+
+
+# -- What's new
+
+WHATS_NEW_URL = (
+    "https://raw.githubusercontent.com/moj-analytical-services/"
+    "platform_user_guidance/master/changelog/whats_new.md"
+)
+
+
+# -- Tool deployments
+
+TOOLS = {
+    "rstudio": {
+        "domain": os.environ.get("RSTUDIO_AUTH_CLIENT_DOMAIN", OIDC_DOMAIN),
+        "client_id": os.environ.get("RSTUDIO_AUTH_CLIENT_ID"),
+        "client_secret": os.environ.get("RSTUDIO_AUTH_CLIENT_SECRET"),
+    },
+    "jupyter-lab": {
+        "domain": os.environ.get("JUPYTER_LAB_AUTH_CLIENT_DOMAIN", OIDC_DOMAIN),
+        "client_id": os.environ.get("JUPYTER_LAB_AUTH_CLIENT_ID"),
+        "client_secret": os.environ.get("JUPYTER_LAB_AUTH_CLIENT_SECRET"),
+    },
+}
+
+# domain where tools are deployed
+TOOLS_DOMAIN = os.environ.get('TOOLS_DOMAIN')
+
+# hostname of NFS server for user homes
+NFS_HOSTNAME = os.environ.get("NFS_HOSTNAME")
+
+
+# -- Async
+
+ASGI_APPLICATION = f"{PROJECT_NAME}.routing.application"
+
+CHANNEL_LAYERS = {
+    'default': {
+        'BACKEND': 'channels_redis.core.RedisChannelLayer',
+        'CONFIG': {
+            'hosts': [
+                (
+                    os.environ.get('REDIS_HOST', 'localhost'),
+                    os.environ.get('REDIS_PORT', '6379'),
+                ),
+            ],
+        },
+    },
+}
+
+# Store Server Sent Events for 24 hours
+EVENTSTREAM_STORAGE_CLASS = 'django_eventstream.storage.DjangoModelStorage'
+
+
+# -- Github
+
+# Allowed Github organizations
+GITHUB_ORGS = [
+    # 'ministryofjustice',
+    'moj-analytical-services',
```

Comment:
We may want to have it an environment variable at some point in the future in case someone else wants to use this.

Reply:
Agreed. I'd suggest leaving that enhancement for another PR - minimize each PR and all that.

Reply:
I agree on having smaller, more focused PRs 😁